test/dynamo/test_aot_autograd.py

```diff
@@ -798,10 +798,11 @@ def _prepare_model_args():
                     continue
                 if min_seq_nr < 0:
                     min_seq_nr = seq_nr
-                mod_name = node.meta.get("source_fn", "")
+                source_fn_stack = node.meta.get("source_fn_stack", [])
```

Comment:
Some questions (maybe for @SherlockNoMad and @jerryzh168?):
- What does source_fn get used for in general?
- Why are we serializing source_fn into the exported graph? Are there use cases for this?

Reply:
source_fn is used mainly for partitioning ops that are getting decomposed right now. Mainly, we'll look for all the ops that were decomposed from a torch.nn.Linear or torch.nn.functional.linear, and partition them into a subgraph.